### PR TITLE
Add a link to TS declaration file to package.json

### LINKED ISSUE #10399: Including declaration in npm package
Please indicate the main declaration file in your package.json:
```javascript
{
  ...,
  "types": "./bundles/dx.all.d.ts",
  ...,
}`

### PR DESCRIPTION
**Pull request by BingoRUS**
Fixes #10399

### CODE REVIEW

**Review by IlyaKhD — APPROVED** [review]

**Comment by ZeBobo5** [comment]
Why this merge is not pushed to 20_1 branch ? An oversight ?

**Comment by BingoRUS** [comment]
@ZeBobo5 We are setting up the 20_1 branch now to start the development of the new DevExtreme version. All recent commits will be pushed there shortly. The PR with package.json changes are waiting for its approval: https://github.com/DevExpress/DevExtreme/pull/10564.

**Pull request merged**
